lib/matplotlib/animation.py

```diff
@@ -1013,6 +1013,16 @@ class to use, such as 'ffmpeg'. If ``None``, defaults to
            on to the `savefig` command which is called repeatedly to
            save the individual frames.
 
+        progress_callback : function, optional
+            A callback function that will be called for every frame to notify the
+            saving progress. It must have the signature ::
+                def func(current_frame: int, total_frames: int) -> Any
+            where *current_frame* is the current frame number and *total_frames*
+            is the total number of frames to be saved. Return values may exist but
+            are ignored.
+            Example code to write the progress to stdout::
+                progress_callback=lambda i, n: print(f'Saving frame {i} of {n}')
```

Comment:
Causes doc-build failure. Needs surrounding empty lines.

Reply:
Thank you for the comments, I'll fix these soon